integration/nav/doc.go

```diff
@@ -0,0 +1,5 @@
+/*
+package nav abstract navigation map related inforamation, such as
+- Location, latitude and longitude for specific point
+*/
+package nav
```

Comment:
Nit: how about `map`? E.g, `map.Location` vs. `nav.Location`.

Reply:
And how about move it into `integration/pkg/api/` folder?

Reply:
Yes, that is the place this package should be.

Reply:
> Nit: how about `map`? E.g, `map.Location` vs. `nav.Location`.

![image](https://user-images.githubusercontent.com/16873751/77674377-bbf5d480-6f48-11ea-8f06-dde6ea27e538.png)